interface/super/edit_globals.php

```diff
@@ -2,179 +2,67 @@
- * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
- *
- * @package OpenEMR
- * @author  Rod Roark <rod@sunsetsystems.com>
- * @author  Brady Miller <brady.g.miller@gmail.com>
- * @link    http://www.open-emr.org
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Rod Roark <rod@sunsetsystems.com>
+ * @author    Brady Miller <brady.g.miller@gmail.com>
+ * @copyright Copyright (c) 2010 Rod Roark <rod@sunsetsystems.com>
+ * @copyright Copyright (c) 2016-2017 Brady Miller <brady.g.miller@gmail.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  */
 
-
-
-
 require_once("../globals.php");
 require_once("../../custom/code_types.inc.php");
 require_once("$srcdir/acl.inc");
 require_once("$srcdir/globals.inc.php");
 require_once("$srcdir/user.inc");
 require_once(dirname(__FILE__)."/../../myportal/soap_service/portal_connectivity.php");
 
-$userMode = (array_key_exists('mode', $_GET) && $_GET['mode'] == 'user');
-
-if (!$userMode) {
-  // Check authorization.
-    $thisauth = acl_check('admin', 'super');
-    if (!$thisauth) {
-        die(xlt('Not authorized'));
-    }
-}
-
-function checkCreateCDB()
-{
-    $globalsres = sqlStatement("SELECT gl_name, gl_index, gl_value FROM globals WHERE gl_name IN
-  ('couchdb_host','couchdb_user','couchdb_pass','couchdb_port','couchdb_dbase','document_storage_method')");
-    $options = array();
-    while ($globalsrow = sqlFetchArray($globalsres)) {
-        $GLOBALS[$globalsrow['gl_name']] = $globalsrow['gl_value'];
-    }
-
-    $directory_created = false;
-    if (!empty($GLOBALS['document_storage_method'])) {
-        // /documents/temp/ folder is required for CouchDB
-        if (!is_dir($GLOBALS['OE_SITE_DIR'] . '/documents/temp/')) {
-            $directory_created = mkdir($GLOBALS['OE_SITE_DIR'] . '/documents/temp/', 0777, true);
-            if (!$directory_created) {
-                echo htmlspecialchars(xl("Failed to create temporary folder. CouchDB will not work."), ENT_NOQUOTES);
-            }
-        }
-
-        $couch = new CouchDB();
-        if (!$couch->check_connection()) {
-            echo "<script type='text/javascript'>alert('".addslashes(xl("CouchDB Connection Failed."))."');</script>";
-            return;
-        }
+use OpenEMR\Admin\Service\AdminMenuBuilder;
+use OpenEMR\Admin\Service\Globals;
+use OpenEMR\Core\Header;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\HttpFoundation\Request;
 
-        if ($GLOBALS['couchdb_host'] || $GLOBALS['couchdb_port'] || $GLOBALS['couchdb_dbase']) {
-            $couch->createDB($GLOBALS['couchdb_dbase']);
-            $couch->createView($GLOBALS['couchdb_dbase']);
-        }
-    }
+$globalsService = new Globals($GLOBALS, new Filesystem());
 
-    return true;
-}
+global $GLOBALS_METADATA;
+global $USER_SPECIFIC_TABS;
+global $USER_SPECIFIC_GLOBALS;
```

Comment:
Are these globals needed anymore? It looks like  you bring them into your functions as parameters.